.github/workflows/ssm-release.yml

```diff
@@ -0,0 +1,249 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+name: release SSM package
+
+on:
+  workflow_dispatch:
```

Comment:
It is possible to make region part of input, so only some region get updated? I am not sure if we will have use cases like that though.

Reply:
Yes. It is a valid use case when deployment fails in some region. The ssm_create.sh will skip regions which have already created SSM package with same version. It is safe to re-run the workflow to continue the failed deployment. So the region name is not required as input.

Reply:
ok, I was thinking if we can use it to test release on specific region, but in that case we can just run the python script locally (against test accounts)